File: Belot.Web/app_ts/models/game.js
var Belot;
(function (Belot) {
    var Web;
    (function (Web) {
        var Models;
        (function (Models) {
            var Game;
            (function (Game) {
                'use strict';
                var CreateRoomModel = (function () {
                    function CreateRoomModel() {
                    }
                    return CreateRoomModel;
                })();
                Game.CreateRoomModel = CreateRoomModel;
                var CreateGameModel = (function () {
                    function CreateGameModel() {
                    }
                    return CreateGameModel;
                })();
                Game.CreateGameModel = CreateGameModel;
            })(Game = Models.Game || (Models.Game = {}));
        })(Models = Web.Models || (Web.Models = {}));
    })(Web = Belot.Web || (Belot.Web = {}));
})(Belot || (Belot = {}));

File: Belot.Backend/Models/Game.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Microsoft.AspNet.Identity.EntityFramework;

namespace Belot.Backend.Models
{
    public class Game
    {
        public int Id { get; set; }

        public ICollection<User> Users { get; set; }

        public ICollection<Score> Score { get; set; }

        public GameMode Mode { get; set; }

        public ICollection<GameCard> PlayedCards { get; set; }

        public int CurrentTurn { get; set; }

        public int ModeFK { get; set; }
    }
}
